_includes/1.0/api/quote.html

```diff
@@ -0,0 +1,161 @@
+                <td>{{ site.format.type.integer }}</td>
+                <td>The delivery status, among :<br />
+                    1: Not Delivered (default)<br />
+                    2: Partially Delivered<br />
+                    3: Delivered</td>
+            </tr>
+            <tr>
+                <td>invoice_status</td>
+                <td>{{ site.format.type.integer }}</td>
+                <td>The invoice status, among :<br />
+                    1: Not invoiced (default)<br />
+                    2: Invoiced<br />
+                    3: Cancelled</td>
+            </tr>
+            <tr>
+                <td>id</td>
+                <td>{{ site.format.type.integer }}</td>
+                <td>Automatically assigned.</td>
+            </tr>
+            <tr>
+                <td>user_code</td>
+                <td>{{ site.format.type.string }}</td>
+                <td>The number assigned to that quote (i.e. QOT-001).</td>
+            </tr>
+            <tr>
+                <td>date</td>
+                <td>{{ site.format.type.date }}</td>
+                <td>The date that will be printed.<br/>Format: {{ site.format.date }}<br/></td>
+            </tr>
+            <tr>
+                <td>currency_iso_code</td>
+                <td>{{ site.format.type.string }}</td>
+                <td>The ISO code of the currency
+                    <a href="http://www.iso.org/iso/catalogue_detail?csnumber=46121" target="_blank" title="ISO 4217">(ISO 4217)</a>.
+                    4 characters max.<br />That currency should have been previously added into your company settings.<br />
+                    Default: your company's currency.
+                </td>
+            </tr>
+            <tr>
+                <td>employee_id</td>
+                <td>{{ site.format.type.integer }}</td>
+                <td>Id of the Employee assigned to that order.<br/></td>
+            </tr>
+            <tr>
+                <td>private_note</td>
+                <td>{{ site.format.type.string }}</td>
+                <td>A private note that won't be printed.</td>
+            </tr>
+            <tr>
+                <td>subtotal</td>
+                <td>{{ site.format.type.double }}</td>
+                <td>Total before taxes and reduce. Based on the content of the order lines.</td>
+            </tr>
+            <tr>
+                <td>reduce_amount</td>
+                <td>{{ site.format.type.double }}</td>
+                <td>Amount of the reduce, based on the subtotal and reduce rate.</td>
+            </tr>
+            <tr>
+                <td>tax_amount</td>
+                <td>{{ site.format.type.double }}</td>
+                <td>Overall tax amount. Includes all taxes, discounts and reduce.</td>
+            </tr>
+            <tr>
+                <td>total</td>
+                <td>{{ site.format.type.double }}</td>
+                <td>The final total the customer needs to pay. All include (taxes, discounts, reduce).</td>
```

Comment:
Since we're on a Quote (devis), it would be more like :
`...total the customer will have to pay`

Reply:
Not resolved